instance/models/openedx_appserver.py

```diff
@@ -208,6 +208,7 @@ def create_configuration_settings(self):
             # querying the Github API twice), we pass it as a context variable.
             'github_admin_username_list': self.github_admin_username_list,
             'newrelic_license_key': settings.NEWRELIC_LICENSE_KEY,
+            'smtp_relay_settings': self.smtp_relay_settings
```

Comment:
@mtyaka I'm wondering if we need to pass these settings as a separate context variable here. According to the comment above, there's a specific reason for doing this for `github_admin_username_list`; but to obtain `smtp_relay_settings` we're just reading configuration variables. So I think it would make sense to remove this line and reference `smtp_relay_settings` as `appserver.smtp_relay_settings` in the template. (That would also make it immediately obvious where the settings come from when reading just the template.)